kernelshark/loong.patch

```diff
@@ -0,0 +1,36 @@
+diff --git a/PKGBUILD b/PKGBUILD
+index 2cc3b9f..8f9f7ea 100644
+--- a/PKGBUILD
++++ b/PKGBUILD
+@@ -36,9 +36,18 @@ provides=(
+   libkshark-plot.so
+   libkshark-gui.so
+ )
+-source=($pkgname-$pkgver.tar.gz::https://git.kernel.org/pub/scm/utils/trace-cmd/kernel-shark.git/snapshot/kernel-shark-$pkgname-v$pkgver.tar.gz)
```

Comment:
Do not directly edit source array, use source+= instead.

See also: https://wszqkzqk.github.io/2024/08/22/loongarchlinux-port-tips/#%E5%86%85%E5%AE%B9%E6%8F%92%E5%85%A5%E7%9A%84%E5%8E%9F%E5%88%99source%E5%93%88%E5%B8%8C%E6%95%B0%E7%BB%84%E7%AD%89

Reply:
Thx, I will fix it soon.